.github/workflows/nightly-playground-deploy.yml

```diff
@@ -130,3 +132,26 @@ jobs:
           else
               echo "dist_version does not belong to 2x or 3x"
           fi
+
+  register-snapshot-repo-enable-alerts:
+    needs: validate-and-deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Register snapshot repo
+        run : |
+          curl -XPUT -f "https://${{needs.validate-and-deploy.outputs.ENDPOINT}}:8443/_snapshot/snapshots-repo" -H 'Content-Type: application/json' -d'
+          {
+            "type": "s3",
+            "settings": {
+              "bucket": "nightly-playgrounds-snapshots-bucket",
+              "region": "us-west-2",
+              "base_path": "${{needs.validate-and-deploy.outputs.PLAYGROUND_ID}}"
+            }
+          }' -u ${{ secrets.OPENSEARCH_USER }}:${{ secrets.OPENSEARCH_PASSWORD }} --insecure
+      # - name: Restore altering configs
+      #   run: |
+      #     curl -XPOST "https://${{needs.validate-and-deploy.outputs.ENDPOINT}}:8443/_snapshot/snapshots-repo/<replace_with_snapshot_id>/_restore" -H 'Content-Type: application/json' -d'
+      #     {
+      #       "indices": ".opendistro-alerting-config,.opensearch-notifications-config",
+      #       "ignore_unavailable": false,
+      #     }' -u ${{ secrets.OPENSEARCH_USER }}:${{ secrets.OPENSEARCH_PASSWORD }} --insecure
```

Comment:
To be enabled after snapshot registration is successful and we have a snapshot to register mentioned indices from